invenio_app_ils/ui/src/invenio_app_ils/common/api/loans/loan.js

```diff
@@ -10,7 +10,7 @@ const get = loanPid =>
     return response;
   });
 
-const getLoanReplaceItemUrl = loanPid => `${loanURL}/${loanPid}/replace-item`;
+const getLoanReplaceItemUrl = loanPid => `${loanURL}${loanPid}/replace-item`;
```

Comment:
Just a comment: I often find myself wondering: "do I have to add the `/` or not?" :) And then... bug :)

What about if we expose all those `url` as functions:
```
getUrl = (loanId='') => {
    return `/circulation/loans/<loanId>`;
}
```

Reply:
`<loanId>` this is not going to work but I get your point :) My personal convention to this super common problem is to always close them on the base urls, and then act accordingly. The double slash doesn't hurt the url but missing one .. will cause issues.